specification/web/resource-manager/Microsoft.CertificateRegistration/stable/2022-09-01/AppServiceCertificateOrders.json

```diff
@@ -0,0 +1,1801 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-09-01",
+    "title": "AppServiceCertificateOrders API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.CertificateRegistration/certificateOrders": {
+      "get": {
+        "tags": [
+          "AppServiceCertificateOrders"
+        ],
+        "summary": "List all certificate orders in a subscription.",
+        "description": "Description for List all certificate orders in a subscription.",
+        "operationId": "AppServiceCertificateOrders_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AppServiceCertificateOrderCollection"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "../../../Microsoft.Web/stable/2022-09-01/CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List App Service Certificate orders by subscription": {
+            "$ref": "./examples/ListAppServiceCertificateOrdersBySubscription.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.CertificateRegistration/validateCertificateRegistrationInformation": {
```

Comment:
>validateCertificateRegistrationInformation [](http://example.com/codeflow?start=81&length=42)

'Information' in the path doesn't seem very clear in meaning. Why not just call this action

validateCertificateRegistration

or

validateCertificateRegistrationOrder? (since it seems like its intended for prevalidating an order)